src/clj_jargon/users.clj

```diff
@@ -55,3 +63,41 @@
   (cond (not (= (.getUserName irodsAccount) (.getProxyName irodsAccount))) true
         (not (= (.getZone irodsAccount) (.getProxyZone irodsAccount))) true
         :else false))
+
+(defn group-exists?
```

Comment:
I decided to put these in clj-jargon.users but if someone would prefer I make a new namespace I'm not opposed, this was just easier (and groups are both users and consist of users, and there's user-focused group listing functions here already).